### DISCUSSION TITLE
Splitting your Plots in many contracts NFT

### DISCUSSION THREAD

**Discussion by MaximusChia** [Chia-Network/chia-blockchain#7425] [Ideas]
This is a courius fact: What happens if you have many NFTs and put a few plots in each one, and then point all your NFts to a single pool?
How the pool would react to this? Each NFT would be considered a different user, (each one with its own dashboard)?
If that so, difficulty will keep low because each NFT has few plots on it?
So would be a kind of cheat? You will be sending proofs so fast on this config.

**Comment by emlowe** [comment]
The difficulty is a multiple - there is no difference between 1 partial @ 10 difficult and 10 partials @ 1 difficulty. You should accumulate points at the same pace regardless of the number of NFTs you have.
